clients/go/cmd/zbctl/internal/commands/failJob.go

```diff
@@ -40,6 +40,7 @@ var (
 	failJobRetriesFlag      int32
 	failJobRetryBackoffFlag time.Duration
 	failJobErrorMessage     string
+	failJobVariablesFlag    string
```

Comment:
AFAIK this isn't a flag

🔧 I think we can rename this to `failJobVariables`

Reply:
> AFAIK this isn't a flag

❓ Hi @korthout. I have a question. Is this different from `completeJob` ? 

https://github.com/camunda/zeebe/blob/7340a588b972bae4d1e33ae04f6673619324440d/clients/go/cmd/zbctl/internal/commands/completeJob.go#L36-L39

Reply:
Ah, that's interesting. My understanding of what a flag is in the context of Cobra appears to be incorrect.

I was expecting flags to be either boolean or not, but in Cobra each argument/option of a command is called a flag. See, https://cobra.dev/#flags.

🔧 Feel free to re-add the flag suffix. But it's fine either way.

Reply:
It's okay, let's merge this now.